test/Microsoft.DotNet.Docker.Tests/ImageTests.cs

```diff
@@ -64,17 +64,17 @@ public static IEnumerable<object[]> GetVerifyImagesData()
             {
                 testData = new List<ImageDescriptor>
                 {
-                    new ImageDescriptor { DotNetCoreVersion = "1.0", PlatformOS = "nanoserver", SdkVersion = "1.1" },
-                    new ImageDescriptor { DotNetCoreVersion = "1.1", PlatformOS = "nanoserver", RuntimeDepsVersion = "1.0" },
-                    new ImageDescriptor { DotNetCoreVersion = "2.0", PlatformOS = "nanoserver" },
+                    new ImageDescriptor { DotNetCoreVersion = "1.0", PlatformOS = "nanoserver-sac2016", SdkVersion = "1.1" },
+                    new ImageDescriptor { DotNetCoreVersion = "1.1", PlatformOS = "nanoserver-sac2016", RuntimeDepsVersion = "1.0" },
+                    new ImageDescriptor { DotNetCoreVersion = "2.0", PlatformOS = "nanoserver-sac2016" },
                     new ImageDescriptor { DotNetCoreVersion = "2.0", PlatformOS = "nanoserver-1709" },
-                    new ImageDescriptor { DotNetCoreVersion = "2.1", PlatformOS = "nanoserver" },
+                    new ImageDescriptor { DotNetCoreVersion = "2.1", PlatformOS = "nanoserver-sac2016" },
                     new ImageDescriptor { DotNetCoreVersion = "2.1", PlatformOS = "nanoserver-1709" },
                 };
 
                 if (OsFilter == null)
```

Comment:
The defaulting of the OsFilter can now be removed.  The change you made will run all the tests but it would be better to just leave the filter not set.